content/techniques/configuration.md

```diff
@@ -44,6 +44,8 @@ DATABASE_USER=test
 DATABASE_PASSWORD=test
 ```
 
+> info **Notice** For a practical example with asynchronous database configuration see the documentation [here](https://docs.nestjs.com/techniques/database#async-configuration)
```

Comment:
I find this sentence kinda confusing for beginners with Nest. A beginner could think "Okay, i'm not using async configuration, so i don't need to have a look there". 

```suggestion
> info **Notice** To access to the configuration service in another registration, have a look in asynchronous database configuration documentation [here](https://docs.nestjs.com/techniques/database#async-configuration)
```
Something like that could be more explicit. My sentence is probably bad, and this is just my point of view, but i think it could be good to clarify the content intended by the developer

Reply:
Let's see if I can find other words to make this warning. Anyway let's see what Kamil thinks.